src/rpc/misc.cpp

```diff
@@ -71,6 +71,7 @@ UniValue getinfo(const UniValue& params, bool fHelp)
             "  \"relayfee\": x.xxxx,         (numeric) minimum relay fee rate for transactions in " + CURRENCY_UNIT + " per 1000 bytes\n"
             "  \"errors\": \"...\"           (string) message describing the latest or highest-priority error\n"
             "  \"errorstimestamp\": \"...\"  (string) timestamp associated with the latest or highest-priority error\n"
+            "  \"lightwalletddisabled\": true|false  (boolean) true if the configuration does not include the lightwalletd extensions\n"
```

Comment:
This PR violates the documented nature of this API:
```
 * @note Do not add or change anything in the information returned by this
 * method. `getinfo` exists for backwards-compatibility only. It combines
 * information from wildly different sources in the program, which is a mess,
 * and is thus planned to be deprecated eventually.
```

If we want to expose what indices are configured, we should do so in a different method (I'd prefer a `zcashd`-specific method personally as this is very `zcashd`-specific).